tasks.py

```diff
@@ -465,6 +465,12 @@ def vscode(context):
     """Launch Visual Studio Code with the appropriate Environment variables to run in a container."""
     command = "code nautobot.code-workspace"
 
+    # Setup PYTHON
+    env_file_path = os.path.join(BASE_DIR, "development/.env")
+    if not os.path.exists(env_file_path):
+        with open(env_file_path, "w") as env_file_obj:
+            env_file_obj.write(f"PYTHON_VER={context.nautobot.python_ver}")
+
     context.run(command, env={"PYTHON_VER": context.nautobot.python_ver})
```

Comment:
I'm curious - after #6659, do we need to also pass `NAUTOBOT_VER` to either of these calls, or is it not needed?

Reply:
Ahh this call is just running `code nautobot.code-workspace` to reopen the current session using the workspace file so I dont really think we need `PYTHON_VER` passed into that at all. It's not like it keeps that environment around in my VSCode session (it doesn't seem to be visible in my terminal session at least)